bindata/network/ovn-kubernetes/common/ipsec-host.yaml

```diff
@@ -34,6 +34,12 @@ spec:
             - matchExpressions:
               - key: network.operator.openshift.io/dpu-host
                 operator: DoesNotExist
+{{ if .RHEL_OS_CHECK }}
```

Comment:
We would not need this if template condition right, as this is never deployed for a hosted cluster.

Reply:
correct, removed it.